### PR TITLE
removed google-analytics scrap and expired time check

### PR DESCRIPTION
**Pull request by TheAngularity**

### CODE REVIEW

**Comment by KiteAnton** [comment]
Sorry to be a bit difficult but can you also rebase/squash commits?  Makes it easier to track related  changes afterwards.

File: main.js [hunk 1]
```diff
@@ -27,21 +27,12 @@ $(document).ready(function () {
...
     }
 
-    // check release time to inform people in case they are running old release
-    if (CONFIGURATOR.releaseDate > (new Date().getTime() - (86400000 * 91))) { // 1 day = 86400000 miliseconds, * 91 = 3 month window
-        console.log('Application version is valid for another: ' + Math.round((CONFIGURATOR.releaseDate - (new Date().getTime() - (86400000 * 60))) / 86400000) + ' days');
-    } else {
-        console.log('Application version expired');
-        GUI.log('You are using an old version of ' + chrome.runtime.getManifest().name + '. There may be a more recent version with improvements and fixes.');
-    }
-
```
Comment (KiteAnton) [inline main.js]: What is the purpose with removing this?

Comment (TheAngularity) [inline main.js]: release date is not maintained correctly ... you should maintain or delete it

last entry:
`'releaseDate': 1463055533515, // new Date().getTime() - Thu May 12 2016 14:19:06 GMT+0200`

**Comment by mikeller** [comment]
I think this is really one for @borisbstyle to answer. He does the releases, and her probably just wasn't aware of `releaseDate` until now. So he should decide if he wants to set it as part of the release process, or if this should go?

**Comment by borisbstyle** [comment]
@mikeller I wasnt aware at all of this very hidden parameter!

**Comment by TheAngularity** [comment]
rebased RP #214

**Pull request closed without merging**